src/UserMenu/UserMenu.js

```diff
@@ -0,0 +1,145 @@
+import React from "react";
+import PropTypes from "prop-types";
+import {
+  Avatar,
+  Divider,
+  IconButton,
+  makeStyles,
+  ListItemIcon,
+  Menu,
+  MenuItem,
+  Typography,
+  withStyles,
+  ListItemText
+} from "@material-ui/core";
```

Comment:
imports not in alphabetical order (not just the mui ones)

Reply:
Have you not written a linter for this yet? 😂